### PR TITLE
Bump to pack 0.26.0

### PR DESCRIPTION
**Pull request by joshwlewis**
There have been a few updates we should pick up (https://github.com/buildpacks/pack/releases/tag/v0.24.0, https://github.com/buildpacks/pack/releases/tag/v0.24.1).

GUS-W-10708989.
GUS-W-11136285.

### CODE REVIEW

**Review by edmorley — APPROVED** [review]

**Comment by joshwlewis** [comment]
Looks like the same CI failure is happening on #280, so it's unlikely to be the pack version change.

**Comment by Malax** [comment]
@joshwlewis Yeah, I haven't looked into the issue yet.

**Comment by Malax** [comment]
Rebased onto main after fixing CI.

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
       - checkout
       - setup_remote_docker
       - pack/install-pack:
-          version: 0.23.0
+          version: 0.24.1
```
Comment (edmorley) [inline .circleci/config.yml:41]: Pack 0.26.0 is now out:
https://github.com/buildpacks/pack/releases/tag/v0.26.0

```suggestion
          version: 0.26.0
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -78,7 +78,7 @@ jobs:
     steps:
       - checkout
       - pack/install-pack:
-          version: 0.23.0
+          version: 0.24.1
```
Comment (edmorley) [inline .circleci/config.yml:81]:
```suggestion
          version: 0.26.0
```

**Comment by Malax** [comment]
Rebased onto main after fixing CI.

**Review by Malax — APPROVED** [review]

**Pull request merged**